CHANGELOG.md

```diff
@@ -4,6 +4,31 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 5.5.0
+### Added
+* Agent: Include `LD_LIBRARY_PATH` if set by command. [#1933](https://github.com/microsoft/onefuzz/pull/1933)
```

Comment:
```suggestion
* Agent: Include `LD_LIBRARY_PATH` if and only if set by command. [#1933](https://github.com/microsoft/onefuzz/pull/1933)
```
Sorry, that [wasn't a typo](https://en.wikipedia.org/wiki/Iff). 😅 

Also, this should be under "Fixed", not "Added". The core functionality was added in `5.4.0`, but the check wasn't handling this env var properly.

Reply:
Oh this makes more sense actually!